launch/tier4_planning_launch/config/scenario_planning/lane_driving/motion_planning/obstacle_velocity_planner/obstacle_velocity_planner.param.yaml

```diff
@@ -0,0 +1,110 @@
+/**:
+  ros__parameters:
+    common:
+      planning_algorithm: "rule_base" # currently supported algorithm is "rule_base"
```

Comment:
"rule_base" may seem confusing.
 Do you have a good name? @yukkysaito

Reply:
or we can say pid_based. We don't use pid controller for obstacle stop though.

Reply:
umm...
pid_based_velocity_decider or pid_base?

Reply:
pid_based is good.

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/570/commits/cae27e51ebbad376a2f0ffb919ac2957a1132c0a
DONE